packages/flutter/lib/src/widgets/scroll_configuration.dart

```diff
@@ -23,7 +24,10 @@ class ScrollBehavior {
   /// The platform whose scroll physics should be implemented.
   ///
   /// Defaults to the current platform.
-  TargetPlatform getPlatform(BuildContext context) => defaultTargetPlatform;
+  TargetPlatform getPlatform(BuildContext context) {
+    return MediaQuery.of(context, nullOk: true)?.platform
```

Comment:
It is significantly over 80 characters. Let me know if you still want to move it to be one line given that.